### ISSUE TITLE
Remove pluralize-mongoose

### ISSUE THREAD

**Issue by ChristianMurphy** [ChristianMurphy/pluralize-mongoose#1]
@vkarpov15 Since Mongoose no longer relies on `pluralize-mongoose`.
Should the repo and the npm package be delisted/removed?

**Comment by vkarpov15** [comment]
Yeah we should just remove it. Sorry I reverted your work, but it's just really tricky to get all the pluralization rules right without breaking somebody's code. I really appreciate all the great work you've done on mongoose so far though :beers:

**Comment by ChristianMurphy** [comment]
> Yeah we should just remove it. Sorry I reverted your work

No problem

> but it's just really tricky to get all the pluralization rules right without breaking somebody's code

Yeah, sorry for causing a snafu :sweat: 

> I really appreciate all the great work you've done on mongoose so far though

Thanks! :smile:
